packages/dd-trace/test/plugins/util/web.spec.js

```diff
@@ -827,4 +860,48 @@ describe('plugins/util/web', () => {
       expect(filtered).to.equal(false)
     })
   })
+
+  describe('obfuscateQs', () => {
```

Comment:
We're going to need better tests than this given the complexity of the default regular expression to make sure it actually works.

Reply:
System tests are for end to end tests, we shouldn't just always rely on them to test things like this. We tend to make them fail a lot too while everything passes in dd-trace, which is a sign that we're missing a lot of tests already.